docs/howtos/setup-android-build-environment.md

```diff
@@ -177,3 +242,23 @@ We now want to configure maven to use the native windows maven repository - then
 * Now you should be ready to roll - `./gradlew install` should complete and publish the components to your native maven repo!
 
 \o/
+
+# Other obsolete content
+
+This content is probably outdated. It should either be removed, or saved!
```

Comment:
I would support just removing this FWIW; if you already know about `publishToMavenLocal` then you don't need these docs, and if you don't know about `publishToMavenLocal` then you're near certain to have a bad time and should instead use the auto-publish flow. I believe the detailed docs on the autopublish flow also have fallback docs describing `publishToMavenLocal`, so we wouldn't be losing much by removing them here.